### ISSUE TITLE
Can't use default reporter in addition to custom reporter

### ISSUE THREAD

**Issue by demonkoryu** [Codeception/CodeceptJS#514]
I'm trying to use the inbuilt reporter in addition to a mocha reporter, e.g. 

`codeceptjs run --reporter mocha-junit-reporter --reporter cli`

As it seems, the inbuilt reporter is only used as a fallback if no --reporter option is given.

It would be nice to have a special reporter name, e.g. "cli", "inbuilt", "default" or whatever for that case.

**Comment by tfiwm** [comment]
I like the idea to extract it because in our case we need the mocha-junit-reporter and another one which shows the process in the jenkins console.

At the moment we use 'mocha-multi-reporters' in combination with the 'mocha-junit-reporter' and the 'mochawesome'.

Maybe this will help you @demonkoryu at the moment ;)

But to have the builtin reporter as a seperated one will be nice than i cloud replace the mochaawesome reporter with the builtin.

**Comment by DavertMik** [comment]
Yes, I tried to make reporters run in parallel but didn't succeed :(

**Comment by tfiwm** [comment]
With mocha-multi-reporters it works great ;)

**Comment by DavertMik** [comment]
Thanks, I will try

**Comment by demonkoryu** [comment]
Thanks, I'm using mocha-multi (tried mocha-multi-reporters before) but  I couldnt use the default CodeceptJS reporter, only the other mocha reporters like spec etc. Any clues as to how I could select the default reporter?
